### PR TITLE
Move analytics logging control from feature flags to settings

### PR DESCRIPTION
**Pull request by orangejenny**
Followup for https://github.com/dimagi/commcare-hq/pull/18814 - no one saw the logged warnings because no one had the logging feature flags turned on. Planning to default this to `warning` for prod and `debug` for staging, we'll see if that's too annoying on staging.

@calellowitz / @biyeun

### CODE REVIEW

**Comment by orangejenny** [comment]
@calellowitz Not especially urgent, would be good to get reviewed by the end of the week.

**Review by snopoke — APPROVED** [review]
One minor comment but not blocking.

File: corehq/apps/analytics/static/analytix/js/logging.js [hunk 1]
```diff
@@ -134,12 +124,14 @@ hqDefine('analytix/js/logging', [
         };
     };
 
+    var levels = ['warning', 'debug', 'verbose'];
     var Level = function (_levelSlug, _levelPrefix, _logger) {
-        var _get = initialAnalytics.getFn('logging'),
+        var logLevel = initialAnalytics.getFn('global')('logLevel'),
+            isVisible = levels.indexOf(_levelSlug) <= levels.indexOf(logLevel),
```
[the remark this answers is not in the document]
Comment (orangejenny) [reply corehq/apps/analytics/static/analytix/js/logging.js]: agreed, fixed

File: corehq/apps/analytics/static/analytix/js/logging.js [hunk 2]
```diff
@@ -126,8 +126,8 @@ hqDefine('analytix/js/logging', [
 
     var levels = ['warning', 'debug', 'verbose'];
     var Level = function (_levelSlug, _levelPrefix, _logger) {
-        var logLevel = initialAnalytics.getFn('global')('logLevel'),
-            isVisible = levels.indexOf(_levelSlug) <= levels.indexOf(logLevel),
+        var globalLevel = initialAnalytics.getFn('global')('globalLevel'),
```
Comment (orangejenny) [inline corehq/apps/analytics/static/analytix/js/logging.js]: Realized this is an error, fixing in 0dcd842

**Pull request merged**